lib/shared/stats.js

```diff
@@ -0,0 +1,81 @@
+"use strict";
+
+module.exports = {
+
+    /**
+     * Start time measurement
+     * @returns {float} t variable for tracking time
```

Comment:
`float` isn't a value that things like VS Code will recognize. Can we change these to `number`?

Reply:
Done! See: https://github.com/eslint/eslint/pull/17850/commits/2d958121bde032fbfe95d24f0058237f7ba01538, https://github.com/eslint/eslint/pull/17850/commits/b671de4a75009d2ff19f9686fd40c08b60e64b45